deployments/chef/README.md

```diff
@@ -0,0 +1,158 @@
+  https://docs.splunk.com/Observability/admin/authentication-tokens/org-tokens.html)
+  to authenticate requests.
+
+- `splunk_realm`: Which Splunk realm to send the data to, e.g. `us1`. The
+  `SPLUNK_REALM` environment variable will be set with this value for the
+  Collector service. This value will derive the `splunk_ingest_url` and
+  `splunk_api_url` attribute values unless they are explicitly set.
+  (**default:** `us0`)
+
+- `splunk_ingest_url`: Explicitly set the Splunk ingest URL, e.g.
+  `https://ingest.us0.signalfx.com`, instead of the URL derived from the
+  `splunk_realm` attribute. The `SPLUNK_INGEST_URL` environment variable will
+  be set with this value for the Collector service. (**default:**
+  `https://ingest.{{ splunk_realm }}.signalfx.com`)
+
+- `splunk_api_url`: Explicitly set the Splunk API URL, e.g.
+  `https://api.us0.signalfx.com`, instead of the URL derived from `splunk_realm`
+  attribute. The `SPLUNK_API_URL` environment variable will be set with this
+  value for the Collector service. (**default:**
+  `https://api.{{ splunk_realm }}.signalfx.com`)
+
+- `collector_version`: Version of the Collector package to install, e.g.
+  `0.34.0`. (**default:** `latest`)
+
+- `splunk_memory_total_mib`: Amount of memory in MiB allocated to the
+  Collector. The `SPLUNK_MEMORY_TOTAL_MIB` environment variable will be set
+  with this value for the Collector service. (**default:** `512`)
+
+- `splunk_ballast_size_mib`: Explicitly set the ballast size for the Collector
+  instead of the value calculated from the `splunk_memory_total_mib` attribute.
+  This should be set to 1/3 to 1/2 of configured memory. The
+  `SPLUNK_BALLAST_SIZE_MIB` environment variable will be set with this value
+  for the Collector service. (**default:** `''`)
+
+- `splunk_bundle_dir`: The path to the [Smart Agent bundle directory](
+  https://github.com/signalfx/splunk-otel-collector/blob/main/internal/extension/smartagentextension/README.md).
+  The default path is provided by the Collector package. If the specified path
+  is changed from the default value, the path should be an existing directory
+  on the node. The `SPLUNK_BUNDLE_DIR` environment variable will be set to
+  this value for the Collector service. (**default:**
+  `/usr/lib/splunk-otel-collector/agent-bundle` on Linux,
+  `%ProgramFiles%\Splunk\OpenTelemetry Collector\agent-bundle` on Windows)
+
+- `splunk_collectd_dir`: The path to the collectd config directory for the
+  Smart Agent bundle. The default path is provided by the Collector package.
+  If the specified path is changed from the default value, the path should be
+  an existing directory on the node. The `SPLUNK_COLLECTD_DIR` environment
+  variable will be set to this value for the Collector service.
+  (**default:** `/usr/lib/splunk-otel-collector/agent-bundle` on Linux,
+  `%ProgramFiles%\Splunk\OpenTelemetry Collector\agent-bundle\run\collectd`
+  on Windows)
```

Comment:
Maybe it'd be clearer to move these to the bottom and the config file paths above the memory options to try to order by likelihood of being updated?